### ISSUE TITLE
description of sequence_science_seq.html

### ISSUE THREAD

**Issue by larnoldgithub** [njcuk9999/apero-drs#779]
https://www.astro.umontreal.ca/~cook/apero-drs/auto/recipe_sequences/spirou/sequence_science_seq.html

says :
Run all nightly science target steps.
This should be run after preprocessing of (at least) all science stars hot star files for all nights being reduced.
Include the following recipes:
EXTTELL: Extracts all science target files   <- shouldn't be EXTOBJ  only ?

**Comment by njcuk9999** [comment]
Yes this is just a typo (section 3 of this page is correct) I'll update it.
